src/workflows/airqo_etl_utils/airqo_utils.py

```diff
@@ -458,11 +458,11 @@ def clean_bam_data(data: pd.DataFrame) -> pd.DataFrame:
     @staticmethod
     def clean_low_cost_sensor_data(data: pd.DataFrame) -> pd.DataFrame:
         data = DataValidationUtils.remove_outliers(data)
+        data.dropna(inplace=True)
```

Comment:
Thanks @NicholasTurner23 for this.
I will approve and also wait for others review notes.